README.md

```diff
@@ -5,18 +5,19 @@ cloudformation-builder
 [![codecov](https://codecov.io/gh/StuPro-TOSCAna/cloudformation-builder/branch/master/graph/badge.svg)](https://codecov.io/gh/StuPro-TOSCAna/cloudformation-builder)
 
 
-CloudFormation-Builder is a Java 8 DSL for creating AWS CloudFormation templates.
+CloudFormation-Builder is a project in Java 8 for creating AWS CloudFormation templates.
 
 Quick Start
 -----------
 
 ### Dependency
 
+Use [jitpack](jitpack.io) to use this project as a maven dependency.
 ```xml
 <dependency>
-    <groupId>com.scaleset</groupId>
+    <groupId>com.github.StuPro-TOSCAna</groupId>
     <artifactId>cloudformation-builder</artifactId>
-    <version>0.2.0</version>
+    <version>(Release you want or hash of any commit)</version>
```

Comment:
Maybe add a placeholder such as {releaseVersion/commitHash} and explain what it should be replaced by below.